.github/workflows/api-pull-request.yml

```diff
@@ -33,7 +35,7 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
     strategy:
-      max-parallel: 4
+      max-parallel: 2
```

Comment:
Why was this reduced to 2?

Reply:
Why should we have more, if we only have 2 Python versions?

Reply:
Oh that's what this is for. I misunderstood. Thanks for the update.